starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    // this.items = listItems;
+    // this.length = listLength;
+    this.items = [];
+    // this.length = 0;
+    this.length = this.items.length;
+  }
+  add(x) {
+    this.items.push(x);
+    this.length++;
+  }
+  get(pos) {
+    var position = this.items[pos - 1];
+    return position;
+  }
+  max() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.max(items);
+  }
+  min() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return Math.min(items);
```

Comment:
because we're in a class we need to use `this.items` otherwise items is undefined. 

to pass this test, you can use a spread operator to return the maximum and minimum ```return Math.min(...this.items);```